source/globalCommands.py

```diff
@@ -1326,7 +1326,8 @@ def script_showGui(self,gesture):
 	def script_review_sayAll(self,gesture):
 		sayAllHandler.readText(sayAllHandler.CURSOR_REVIEW)
 	# Translators: Input help mode message for say all in review cursor command.
-	script_review_sayAll.__doc__ = _("Reads from the review cursor up to end of current text, moving the review cursor as it goes")
+	script_review_sayAll.__doc__ = _("Reads from the review cursor up to the end of the current text, \

```

Comment:
Rather than using '\' to split these lines, you can just have a string on each line. Strings are automatically joined in line continuations like this. The brackets groups them and means they will be joined together. Though note the space at the star of the first string.

``` python
	script_review_sayAll.__doc__ = _(
		"Reads from the review cursor up to the end of the current text,"
		" moving the review cursor as it goes"
	)
```

Also note that the first string starts on a new line after the opening bracket, this is "hanging indent" style, and will keep the lint checks happy

Reply:
Thanks very much, this is a good idea.
> Though note the space at the star of the first string.
You mean the last string?

One more question. If I split the line after the bracket, will this not cause errors in the gettext function for translators? After some research my understanding was that I could split like this, including the _ in the string:

	script_review_sayAll.__doc__ = (
		"_(Reads from the review cursor up to the end of the current text,"
		" moving the review cursor as it goes")
	)

But if this does not have an impact, then I would prefer your aproach.

Reply:
@feerrenrut I think I have addressed all the review actions here. Could you please check the https://github.com/nvaccess/nvda/pull/10130#discussion_r317332023 ? Especially the hanging indent in the input message? I am not sure if the implementation as it is now in this PR would affect the translation.

Reply:
I believe that the `checkPot` test (which runs as part of a PR build) should catch any errors here. If you are uncertain, check for the string in the created `.pot` file (after merging master, it will be in the artifacts tab for your PR build). Putting the `_(` inside the string will not work, since the underscore or "line" is actually just a short name for a function that looks up the translation.